documentation/asciidoc/topics/proc_shut_down_gracefully.adoc

```diff
@@ -1,5 +1,15 @@
 [id='shutting_down-{context}']
-= Gracefully Shutting Down {brandname} Clusters
+= Shutting Down {brandname} Clusters
+Shutting down {cacheservice} nodes removes all data in the cache. For
+{datagridservice} nodes, you should configure the storage size for
+{datagridservice} nodes to ensure that the persistent volume can hold all your
+data.
+
+If the available container storage is less than the amount of memory available
+to {datagridservice} nodes, {brandname} writes an exception to logs and data
```

Comment:
Operator will write a following message to the log: **WARNING: persistent volume size is less than memory size. Graceful shutdown may not work.** We are planning to send events like this to the event queue instead of logs.